### PR TITLE
add add and sub of RistrettoPoint and Ciphertext

### PR DESCRIPTION
**Pull request by contrun**
Let $g$ be a base point, $x$ and $y$ be two public keys, $m_1$ and $m_2$ be two ristretto points.
We can view a RistrettoPoint $(m_1)$ equivalent to Ciphertext $(m_1 + 0x, 0g)$, and define the subtraction of RistrettoPoint and Ciphertext as
$(0g, m_1 + 0x) - (rg, m_2 + ry) = (rg, m_1 - m_2 - ry)$.
We can decipher the ciphertext with the secret key of the original ciphertext. This is can be useful in some case, for example, we can calculate the subtraction of plaintext transaction fee and ciphertext balance. The resulting ciphertext can be deciphered with the original secret key.

Edit: fix the minus sign.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -706,4 +757,38 @@ mod tests {
...
+        let plaintext = RistrettoPoint::random(&mut csprng);
+        let ciphertext = pk.encrypt(plaintext);
+        let plaintext2 = RistrettoPoint::random(&mut csprng);
+
+        println!(
+            "1: {:?}, 2: {:?}, 3: {:?}",
+            sk.decrypt(plaintext2 + ciphertext),
+            sk.decrypt(ciphertext + plaintext2),
+            plaintext + plaintext2
+        );
```
Comment (iquerejeta) [inline src/lib.rs]: This should be removed.  To maintain the consistency of not printing anything in the tests. This is the only change I see necessary.

**Review by iquerejeta — CHANGES_REQUESTED** [review]
While the description of the PR has typos in the signs (missing '-'), the code is correct. Only thing is the line printing

File: src/lib.rs (hunk 1 above)
Comment (contrun) [reply src/lib.rs]: Thanks. I've already removed it.

**Review by iquerejeta — APPROVED** [review]

**Comment by iquerejeta** [comment]
I've updated the version in crates.io to v0.1.3, where these functionalities are included

**Pull request merged**